lib/_stream_readable.js

```diff
@@ -399,7 +400,13 @@ function howMuchToRead(n, state) {
 // You can override either this method, or the async _read(n) below.
 Readable.prototype.read = function(n) {
   debug('read', n);
-  n = parseInt(n, 10);
+  // Same as parseInt(undefined, 10), however V8 7.3 performance regressed
+  // in this scenario, so we are doing it manually.
+  if (n === undefined) {
+    n = NaN;
+  } else if (typeof n !== 'number') {
+    n = parseInt(n, 10);
```

Comment:
This deviates a tiny bit from the former behavior: decimal numbers where coerced to integers. Is there any performance difference when using `Number.isInteger(n)`?

Reply:
No perf regression using `Number.isInteger(n)`. Updating.